module/solutions/helper.js

```diff
@@ -1693,6 +1693,72 @@ module.exports = class SolutionsHelper {
     });
   }
 
+
+  /**
+   * Verify solution id
+   * @method
+   * @name verifySolution
+   * @param {String} solutionId - solution Id.
+   * @param {String} userId - user Id.
+   * @param {String} userToken - user token.
+   * @param {Boolean} createProject - create project.
+   * @param {Object} bodyData - Req Body.
+   * @returns {Object} - Details of the solution.
+   */
+
+   static verifySolution(solutionId = "", bodyData = {}) {
```

Comment:
@ankitpws - Please add some comments on all other function call and what it does and returns